phpstan.neon.dist

```diff
@@ -15,7 +15,7 @@ parameters:
     # run `php src/Core/DevOps/StaticAnalyze/phpstan-bootstrap.php` to create the container
     symfony:
         constantHassers: false
-        containerXmlPath: 'var/cache/phpstan_dev/Shopware_Core_DevOps_StaticAnalyze_StaticAnalyzeKernelPhpstan_devDebugContainer.xml'
+        containerXmlPath: 'var/cache/static_phpstan_dev/Shopware_Core_DevOps_StaticAnalyze_StaticAnalyzeKernelPhpstan_devDebugContainer.xml'
```

Comment:
is this change needed? If so do we need to adjust `phpstan-bootstrap`file as well? as that thing created the folder or where is the static_ prefix coming from? 🤔

Reply:
@keulinho basically in that test `\Shopware\Tests\Integration\Core\Framework\Plugin\PluginManagementServiceTest` it creates a new static kernel and it gets created in `test` dir of cache directory, which is the same dir as the integration tests are running in, so when it deletes the directory in `tearDown` it also deletes the cache for the current running test suite. I suppose we could just update the env in the test to scope cache directory, but it might be a might fool proof for whatever else might use it in the future. My fix with rebooting fixes the specific issue I have, but I think this could cause other issues in the future

Reply:
we don't need to update `phpstan-bootstrap.php` we just add a static prefix on the cache dir and then reference that in config file above.